internal/stage_common.go

```diff
@@ -58,14 +59,18 @@ func dumpResponse(logger *logger.Logger, resp *http.Response) error {
 	if err != nil {
 		return fmt.Errorf("Failed to dump rsponse: '%v'", err)
 	}
-	logger.Infof("Received response: (status line) %s", getFirstLine(string(respDump)))
 	logPrefix := ">>>"
 	logger.Debugf("Received response: (Messages with %s prefix are part of this log)", logPrefix)
 	logFriendlyHTTPMessage(logger, string(respDump), logPrefix)
 
 	return nil
 }
 
+func logCurl(logger *logger.Logger, req *http.Request) {
+	logger.Infof("You can use the following curl command to test this locally")
+	logger.Infof("> %s", httpRequestToCurlString(req))
```

Comment:
```suggestion
	logger.Infof("$ %s", httpRequestToCurlString(req))
```

Would remove the "You can use the following curl command" line, might be noisy and I think it's obvious if we do `$` for the command